main.py

```diff
@@ -2,7 +2,9 @@
 from shedlib.Parsing.parsing import Parser as Parser
 from shedlib.Time.Time import Time as Time
 import time
+import os
 
+__VERSION__ = "0.1.1"
```

Comment:
Well, the problem is that now we need to modify __VERSION__ variable at main.py each time we move to new version of app